### PR TITLE
Some updates and some answers variety. 2.0

### PR DESCRIPTION
**Pull request by Anonymioo**
Latest changes. 
Idk why but `Atom` made useless 'additions and deletions' of spaces.

### CODE REVIEW

File: Monika After Story/game/chess.rpy [hunk 1]
```diff
@@ -179,7 +179,9 @@ init 1 python in mas_chess:
         "Tch-",
         "Winning isn't everything, you know...",
         "Ahaha,{w} I was just letting you win since you keep losing so much.",
-        "Oh, you won.{w} I should have taken this game seriously, then."
+        "Oh, you won.{w} I should have taken this game seriously, then.",
```
Comment (Rai99) [inline Monika After Story/game/chess.rpy]: First of all, I'm not sure if any of these are even still used. 

Second, I really don't think we want Monika to be this mean about a normal game of chess.

File: Monika After Story/game/chess.rpy [hunk 2]
```diff
@@ -179,7 +179,9 @@ init 1 python in mas_chess:
...
+        "I was just easy on you.",
+        "Did you used a{w} 'Solver'{w} or you're cheating?"
```
Comment (Rai99) [inline Monika After Story/game/chess.rpy]: Replace "used" with "use". 

Did you mean to put waits on both sides of "'Solver'"?

Also, that's pretty mean for her to accuse you of cheating just when you win.

File: Monika After Story/game/chess.rpy [hunk 3]
```diff
@@ -179,7 +179,9 @@ init 1 python in mas_chess:
         "Tch-",
         "Winning isn't everything, you know...",
         "Ahaha,{w} I was just letting you win since you keep losing so much.",
-        "Oh, you won.{w} I should have taken this game seriously, then."
+        "Oh, you won.{w} I should have taken this game seriously, then.",
+        "I was just easy on you.",
```
Comment (Rai99) [inline Monika After Story/game/chess.rpy]: I really don't think we want to trivialize the player's chess victories against Monika any further.

File: Monika After Story/game/chess.rpy [hunk 4]
```diff
@@ -196,7 +198,8 @@ init 1 python in mas_chess:
     # 1k expressions
     _monika_wins_line_quips = (
         "Ahaha, do you even know how to play chess?", # use this for surrenders too
-        "Are you {i}that{/i} bad? I wasn't even taking this game seriously."
+        "Are you {i}that{/i} bad? I wasn't even taking this game seriously.",
```
Comment (Rai99) [inline Monika After Story/game/chess.rpy]: Again, we probably don't want Monika to be mean about chess.

File: Monika After Story/game/script-affection.rpy [hunk 5]
```diff
@@ -938,7 +945,8 @@ init 15 python in mas_affection:
             "...Fine.",
             "I guess we can play a game.",
             "I guess, if you really want to.",
-            "I suppose a game would be fine."
+            "I suppose a game would be fine.",
+            "...Yes...why not?"
```
Comment (multimokia) [inline Monika After Story/game/script-affection.rpy]: also informality thing here
```suggestion
            "...{w=0.5}yeah, why not?"
```

**Pull request merged**